subiquity/controllers/zdev.py

```diff
@@ -633,7 +637,15 @@ def typeclass(self):
 
 class ZdevController(SubiquityController):
 
+    autoinstall_key = "s390x-devices"
```

Comment:
zdev would be better, as it is `lszdev` `chzdev` not `s390x-devices`.

Reply:
OK.